e2e/tests/SSR/index.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect, test } from '@playwright/test';
+import { exec, type ChildProcessWithoutNullStreams } from 'child_process';
+import { readFileSync, watch } from 'fs';
+import path from 'path';
+import setupCoverage from '../../setup/setupCoverage';
+
+setupCoverage();
+
+test.describe('SSR', () => {
+  const PORT = 3000;
+  const nextTemplateDir = path.resolve(
+    process.cwd(),
+    '.codesandbox',
+    'templates',
+    'next'
+  );
+
+  let task: ChildProcessWithoutNullStreams;
+
+  test.beforeAll(({}, testInfo) => {
+    testInfo.setTimeout(60 * 1000);
+    task = exec(`npm start next -- -p ${PORT} --no-watch`, {
+      cwd: process.cwd(),
+    });
+    task.stdout.pipe(process.stdout);
+    task.stderr.pipe(process.stderr);
+    return new Promise<void>((resolve) => {
+      const watcher = watch(nextTemplateDir, (event, filename) => {
+        if (filename === '.instrumentation') {
+          watcher.close();
+          resolve();
+        }
+      });
+    });
+  });
+
+  test.afterAll(() => task.kill());
+
+  test('SSR with Next.js', async ({ page }) => {
+    await page.goto(`http://localhost:${PORT}/`);
+    await page.waitForLoadState('load');
+    await test.step('edit text', async () => {
+      await page
+        .locator('canvas')
+        .nth(1)
+        .dblclick({
+          position: {
+            x: 625,
+            y: 50,
+          },
+        });
+      await page.getByRole('textbox').press('Control+a');
+      await page
+        .getByRole('textbox')
+        .fill("fabric supports SSR!\nIsn't that amazing?!");
```

Comment:
This message is confusing.
What fabric support is only not blowing up node with an import that run browser code.
How do we ensure that the page was actually rendered by server?
Imho the test can just curl the output of  http://localhost:${PORT}/ and verify it contains the snapshot of html.
The fact that the page load could also simply mean that SSR isn't active, no?

I would also change `fabric supports SSR!\nIsn't that amazing?!` in a softer `a page that uses fabricJS will render ok with framework that use SSR`

Reply:
> How do we ensure that the page was actually rendered by server?

It doesn't, does it?
That is truly supporting SSR.
It is confusing saying the current state is SSR support.
I mean you can import fabric in SSR and it will not break. It will run on the browser after dehydration.
It is interesting to support SSR by rendering on node and sending it as an image or something.

Reply:
Regarding the test I just want to prove fabric works after window loads